app/app.go

```diff
@@ -114,7 +118,7 @@ var (
 		mint.AppModuleBasic{},
 		distr.AppModuleBasic{},
 		gov.NewAppModuleBasic(
-			paramsclient.ProposalHandler, distrclient.ProposalHandler, upgradeclient.ProposalHandler, upgradeclient.CancelProposalHandler,
+			append(wasmclient.ProposalHandlers, paramsclient.ProposalHandler, distrclient.ProposalHandler, upgradeclient.ProposalHandler, upgradeclient.CancelProposalHandler)...,
```

Comment:
why not just
```suggestion
			wasmclient.ProposalHandlers, paramsclient.ProposalHandler, distrclient.ProposalHandler, upgradeclient.ProposalHandler, upgradeclient.CancelProposalHandler,
```

?

Reply:
`wasmclient.ProposalHandlers` is an array, where the others are not.

Reply:
Then you can do following:
```
gov.NewAppModuleBasic(
    paramsclient.ProposalHandler, distrclient.ProposalHandler, upgradeclient.ProposalHandler, upgradeclient.CancelProposalHandler, ...wasmclient.ProposalHandlers)
```
Note: this will work only if the `wasmclient.ProposalHandlers` won't override the previous options. 
If you are not sure then probably it's better to do as you did.

Reply:
Tried this as well not working